crypto/merkle/proof.go

```diff
@@ -50,9 +50,6 @@ func ProofsFromByteSlices(items [][]byte) (rootHash []byte, proofs []*Proof) {
 // Verify that the Proof proves the root hash.
 // Check sp.Index/sp.Total manually if needed
 func (sp *Proof) Verify(rootHash []byte, leaf []byte) error {
-	if rootHash == nil {
-		return fmt.Errorf("invalid root hash: cannot be nil")
-	}
```

Comment:
Want to verify this is okay, this was not from the PR but was a part of the diff between this and the branch the original feat was merged to.